test/e2e/pipeline/src/test/java/org/apache/shardingsphere/test/e2e/data/pipeline/cases/base/PipelineBaseE2EIT.java

```diff
@@ -108,11 +108,12 @@ public abstract class PipelineBaseE2EIT {
     
     private Thread increaseTaskThread;
     
-    public PipelineBaseE2EIT(final PipelineTestParameter testParam) {
-        databaseType = testParam.getDatabaseType();
+    public PipelineBaseE2EIT(final PipelineTestParameter testParameter) {
```

Comment:
Based on Code of Conduct, `testParameter` could be `testParam`, so it's not necessary to rename it.